scripts/merge_llama2_with_chinese_lora_low_mem.py

```diff
@@ -256,18 +272,17 @@ def merge_shards(output_dir, num_shards: int):
         print("Cannot find lora model on the disk. Downloading lora model from hub...")
         base_model_path = snapshot_download(repo_id=base_model_path)
     ckpt_filenames = sorted([f for f in os.listdir(base_model_path) if re.match('pytorch_model-(\d+)-of-(\d+).bin',f)])
```

Comment:
The ckpt filename of 1.3B model is `pytorch_model.bin`, which will not match `'pytorch_model-(\d+)-of-(\d+).bin'`. Add a judgement here whether base model size is 1.3B

Reply:
Fixed